jib-core/src/main/java/com/google/cloud/tools/jib/global/JibSystemProperties.java

```diff
@@ -27,12 +27,13 @@
   @VisibleForTesting
   public static final String SEND_CREDENTIALS_OVER_HTTP = "sendCredentialsOverHttp";
 
+  @VisibleForTesting
+  static final String USE_CROSS_REPOSITORY_BLOB_MOUNTS = "jib.useCrossRepositoryBlobMounts";
```

Comment:
Maybe this can be shortened to just `jib.useBlobMounts`? (There's currently no other types of blob mounts)